openqaoa/workflows/optimizer.py

```diff
@@ -567,6 +666,7 @@ def __init__(self, device=DeviceLocal("vectorized")):
         """
         super().__init__(device)
         self.circuit_properties = CircuitProperties()
+        self.algorithm = 'qaoa'
```

Comment:
why to add the attribute `self.algorithm`? The same information is in `self.header["algorithm"]`

Reply:
Because before we did not have it! 

Indeed, I think now we can now remove them